src/styles/containers/_app.scss

```diff
@@ -0,0 +1,30 @@
+// container for menus and content
+.app {
+  height: 100vh;
+  perspective: 1500px;
+  width: 100vw;
+
+  // page wrap for content
+  &__content {
+    background: $navy-taupe;
```

Comment:
I want to try and deprecate these color vars in favor of a map set reference. so instead of `background: $navy-taupe;` let's use `background: palette('background')` instead.